src/azul/chalice.py

```diff
@@ -665,6 +665,13 @@ def retries(self) -> Iterator[retry]:
                 else:
                     yield retry.bind(self, handler_name)
 
+    @property
+    def tf_function_resource_names(self) -> Iterator[str]:
+        yield self.unqualified_app_name
+        for handler_name, handler in self.handler_map.items():
+            if isinstance(handler, chalice.app.EventSourceHandler):
```

Comment:
What's the handler when this is False?

Reply:
function AzulChaliceApp.default_routes.<locals>

swagger_redirect
swagger_ui
swagger_initializer
swagger_resource
openapi
version
robots_txt

function HealthApp.default_routes.<locals>

health
basic_health
cached_health
fast_health
custom_health

function oauth2_redirect.default_routes.<locals>

oauth2_redirect

function list_catalogs

list_catalogs

function HealthApp.default_routes.<locals>

post_notification

function SchemaController.default_routes.<locals>

get_schema

...